index.html

```diff
@@ -5,5 +5,6 @@
   <body>
     <h1> Welcome to the home page</h1>
     <h2> How you doing </h2>
+    <p> Adding content into the paragraph tag </p>
```

Comment:
Hurray...! , You made the nice change , Please also add some other tags

Reply:
Okay  thanks for confirmation ...